errbot/core_plugins/chatRoom.py

```diff
@@ -33,12 +33,15 @@ def callback_connect(self):
                     self.log.exception("Joining room %s failed", repr(room))
 
     def _join_room(self, room):
-        room_name = compat_str(room)
-        if room_name is not None:
-            room, username, password = (room_name, self.bot_config.CHATROOM_FN, None)
+        if isinstance(room, tuple):
```

Comment:
Ah, ok. I saw only tuples in config template so that's what I thought can ever exist:
https://github.com/errbotio/errbot/blob/master/errbot/config-template.py#L149